.github/workflows/load_tests.yml

```diff
@@ -16,12 +18,13 @@ jobs:
         uses: k6io/action@v0.1
         with:
           filename: k6LoadTests.js
-      - name: k6 Load Tests failed notifications
+      - name: k6 Load Tests notifications
         uses: rtCamp/action-slack-notify@v2
         env:
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
           SLACK_USERNAME: AlertManager
+          SLACK_COLOR: ${{ job.status == 'success' && 'good' || (job.status == 'failure' && 'danger' || 'warning') }}
```

Comment:
This seems to be recently fixed: https://github.com/rtCamp/action-slack-notify/issues/77

Reply:
seems so. the main problem with schedule right now. first of all it seems that pr has to be merged so that schedule changes are updated. so its not so easy to test, as on push notifications are working, on schedule gha skips the notifications at all (you can see from the action history details.

Reply:
I had same type of problem, this might help:

```
      - name: Run k6 tests
        uses: k6io/action@v0.1
        id :k6
        with:
          filename: k6LoadTests.js
      - name: k6 Load Tests failed notifications
      - name: k6 Load Tests notifications
        uses: rtCamp/action-slack-notify@v2
        env:
          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
          SLACK_USERNAME: AlertManager
          SLACK_COLOR: ${{ steps.k6.outcome == 'success' && 'good' || (steps.k6.outcome == 'failure' && 'danger' || 'warning') }}
            if: always() && (steps.k6.outcome == 'success' || steps.k6.outcome == 'failure')
          ```